rules/downgrade-php80/tests/Rector/Property/DowngradeUnionTypeTypedPropertyRector/DowngradeUnionTypeTypedPropertyRectorTest.php

```diff
@@ -11,7 +11,7 @@
 use Symplify\SmartFileSystem\SmartFileInfo;
 
 /**
- * @requires PHP >= 8.0
+ * @requires PHP = 8.1
```

Comment:
Why this change? It skips the test until 8.1, but it should work on 8.0.
Same for other conditions

Reply:
The test currently failure, see https://github.com/rectorphp/rector/pull/4779#discussion_r536491712 , this PR is for enabing PHP 8.0 in CI, the fix should be in different PR I think

Reply:
I see, ok then :+1: